app/src/views/CountryPreparedness/PreviousAssessmentChart/styles.module.css

```diff
@@ -2,7 +2,7 @@
     height: 20rem;
 
     .path {
-        stroke: var(--go-ui-color-primary-red);
+        stroke: #236192;
```

Comment:
@frozenhelium shouldn't we use a design token for this as well?

Reply:
Changed; it was missed.